packages/frontend-api/src/Model/Resolver/Products/ProductResolverMap.php

```diff
@@ -4,190 +4,93 @@
-    }
-
-    /**
-     * @required
-     * @param \Shopsys\FrontendApiBundle\Model\Product\ProductAccessoryFacade $productAccessoryFacade
-     * @internal This function will be replaced by constructor injection in next major
-     */
-    public function setProductAccessoryFacade(ProductAccessoryFacade $productAccessoryFacade): void
-    {
-        if ($this->productAccessoryFacade !== null && $this->productAccessoryFacade !== $productAccessoryFacade) {
-            throw new BadMethodCallException(
-                sprintf('Method "%s" has been already called and cannot be called multiple times.', __METHOD__)
-            );
-        }
-        if ($this->productAccessoryFacade !== null) {
-            return;
-        }
-
-        @trigger_error(
-            sprintf(
-                'The %s() method is deprecated and will be removed in the next major. Use the constructor injection instead.',
-                __METHOD__
-            ),
-            E_USER_DEPRECATED
-        );
-        $this->productAccessoryFacade = $productAccessoryFacade;
-    }
-
-    /**
-     * @required
-     * @param \Shopsys\FrameworkBundle\Model\Customer\User\CurrentCustomerUser $currentCustomerUser
-     * @internal This function will be replaced by constructor injection in next major
-     */
-    public function setCurrentCustomerUser(CurrentCustomerUser $currentCustomerUser): void
-    {
-        if ($this->currentCustomerUser !== null && $this->currentCustomerUser !== $currentCustomerUser) {
-            throw new BadMethodCallException(
-                sprintf('Method "%s" has been already called and cannot be called multiple times.', __METHOD__)
-            );
-        }
-        if ($this->currentCustomerUser !== null) {
-            return;
-        }
-
-        @trigger_error(
-            sprintf(
-                'The %s() method is deprecated and will be removed in the next major. Use the constructor injection instead.',
-                __METHOD__
-            ),
-            E_USER_DEPRECATED
-        );
-        $this->currentCustomerUser = $currentCustomerUser;
+        $this->productEntityFieldMapper = $productEntityFieldMapper;
+        $this->productArrayFieldMapper = $productArrayFieldMapper;
     }
 
     /**
      * @required
-     * @param \Shopsys\FrameworkBundle\Model\Product\ProductFacade $productFacade
+     * @param \Shopsys\FrontendApiBundle\Model\Resolver\Products\DataMapper\ProductArrayFieldMapper $productArrayFieldMapper
      * @internal This function will be replaced by constructor injection in next major
      */
-    public function setProductFacade(ProductFacade $productFacade): void
+    public function setProductArrayFieldMapper(ProductArrayFieldMapper $productArrayFieldMapper): void
     {
-        if ($this->productFacade !== null && $this->productFacade !== $productFacade) {
+        if ($this->productArrayFieldMapper !== null && $this->productArrayFieldMapper !== $productArrayFieldMapper) {
```

Comment:
I know it is ok for our standards as it fits so far but could you split it to multiline as it is the way how we are using it almost everywhere?

Reply:
okay, in 01a4cd41fa1b21027952092c8132130c5e95ab78